perun-core/src/test/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntryIntegrationTest.java

```diff
@@ -885,7 +905,7 @@ public void updateResource() throws Exception {
 		facility = setUpFacility();
 		resource = setUpResource();
 
-		Resource resourceToUpdate = resourcesManager.createResource(sess, resource, vo, facility);
```

Comment:
I not sure, if this test does anything useful (wasn't before either). It should create 2 resources, which are different, then change one of them and see, it property was really updated. Also now it should check (maybe in another test method) that you can't update resource name to the one, that already exists. Update method itself returns updated object and it should match getById like method results. Can you please rewrite this test. Thank you.